tests/test_utils.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime
+
+import pytest
+
+from devlog.utils.blog import post_from_markdown
+from devlog.utils.pagination import get_page
+
+
+def test_pagination_invalid_param(client, app):
+    val = 'dummy'
+    with app.test_request_context(f'/?p={val}'):
+        page = get_page()
+        assert page != val
+        assert page == 1
+
+
+@pytest.mark.usefixtures('app')
+class TestPostImport:
+
+    def test_import_ok(self):
+        md = '''---
+title: Wpis testowy
+date: 2020-06-21T18:24:12
+tags:
+  - tag1
+  - tag2
+author: Test Testowy
+---
+
+Pierwsza linijka tekstu, zawierająca *markup*.
+
+<!-- more -->
+
+Druga linijka tekstu, zawierająca *markup*.
+'''
+        rv = post_from_markdown(md)
+        assert rv.title == 'Wpis testowy'
+        assert '<em>' in rv.summary
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/jarek.zgoda/devlog/pullRequest?prid=5774733)